apis_core/collections/models.py

```diff
@@ -4,6 +4,21 @@
 from apis_core.generic.abc import GenericModel
 
 
+class SkosCollectionManager(models.Manager):
+    def get_by_full_path(self, name: str):
+        """
+        Return a collection specified by its full path, from the root colletion
+        to the leaf collection, delimited by `|`. I.e. if there is a collection
+        named `foo` and it has a parent named `bar` and `bar` does not have a
+        parent, then you can use the string "bar|foo" to get the `foo` collection.
+        """
+        names = name.split("|")
+        parent = None
+        while names:
+            parent = self.get(parent=parent, name=names.pop(0))
+        return parent
```

Comment:
```suggestion
            collection = self.get(parent=collection, name=names.pop(0))
        return collection
```
Might be a minor thing, but the function is actually not returning the parent, but the leaf. So I suggest to change the variable name.

Reply:
I understand the concern and its a valid point, but the thing is that it is a `while` loop and the collection item is always the result **and** the parent of the next iteration. Only after the last iteration its not a parent anymore